classifiers/nlp/ZeroShotTFClassifier/Dockerfile

```diff
@@ -1,28 +1,15 @@
-FROM continuumio/miniconda3 as conda
-
-COPY requirements.txt /
-RUN conda update conda -c conda-forge && \
-    pip install -r requirements.txt --no-cache-dir && \
-    conda clean -afy
-
-FROM nvidia/cuda:11.0-base-ubuntu20.04 as base
```

Comment:
Why do we remove these?

Reply:
It was impossible to make it work with the `tensorflow` conficts.

So if we want to keep the versions rolling we need to remove

Reply:
oh ok